src/helm/config/model_metadata.yaml

```diff
@@ -1765,7 +1765,16 @@ models:
     tags: [TEXT_MODEL_TAG, OPENAI_CHATGPT_MODEL_TAG, LIMITED_FUNCTIONALITY_TEXT_MODEL_TAG, INSTRUCTION_FOLLOWING_MODEL_TAG]
 
   - name: openai/gpt-4-vision-preview
-    display_name: GPT-4V (preview)
+    # According to https://platform.openai.com/docs/models/gpt-4-turbo-and-gpt-4, this model has pointed gpt-4-1106-vision-preview.
+    display_name: GPT-4V (1106 preview)
```

Comment:
Date GPT-4V preview

Reply:
This is going to get out of date in the future... "GPT-4V (preview)" is better.